package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "inclusivity-slack-inviter",
+  "version": "0.0.0",
+  "description": "Automated Slack invitations for the Node.js Inclusivity WG",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
I'm pretty certain we use an implicit DCO everywhere including here, but @mikeal would be able to answer the specifics better than I can.